bindings/python/iota_sdk/types/context_input.py

```diff
@@ -22,11 +23,31 @@ class ContextInput():
     type: int
 
 
+@json
+@dataclass
+class CommitmentContextInput(ContextInput):
+    """A Commitment Input allows referencing a commitment to a certain slot.
+
+    Attributes:
+        type: The type of context input.
+        commitment_id: The commitment identifier to reference to.
```

Comment:
This comment makes sense but is poorly worded

Reply:
We addressing this?

Reply:
Probably just "The commitment identifier" is fine, right?